docs/modules/clusters/pages/phi-accrual-detector.adoc

```diff
@@ -68,24 +37,39 @@ YAML::
 ----
 hazelcast:
   properties:
-    hazelcast.heartbeat.failuredetector.type: phi-accrual
-    hazelcast.heartbeat.interval.seconds: 1
-    hazelcast.max.no.heartbeat.seconds: 60
-    hazelcast.heartbeat.phiaccrual.failuredetector.sample.size: 200
-    hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis: 100
+    hazelcast.heartbeat.failuredetector.type: phi-accrual <1>
+    hazelcast.heartbeat.interval.seconds: 1 <2>
+    hazelcast.max.no.heartbeat.seconds: 60 <3>
+    hazelcast.heartbeat.phiaccrual.failuredetector.threshold: 10 <4>
+    hazelcast.heartbeat.phiaccrual.failuredetector.sample.size: 200 <5>
+    hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis: 100 <6>
 ----
-====
-
-**Programmatic Configuration:**
 
+Java Member API::
++
 [source,java]
 ----
 Config config = ...;
-config.setProperty("hazelcast.heartbeat.failuredetector.type", "phi-accrual");
-config.setProperty("hazelcast.heartbeat.interval.seconds", "1");
-config.setProperty("hazelcast.max.no.heartbeat.seconds", "60");
-config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.threshold", "10");
-config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.sample.size", "200");
-config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis", "100");
+config.setProperty("hazelcast.heartbeat.failuredetector.type", "phi-accrual"); <1>
+config.setProperty("hazelcast.heartbeat.interval.seconds", "1"); <2>
+config.setProperty("hazelcast.max.no.heartbeat.seconds", "60"); <3>
+config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.threshold", "10"); <4>
+config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.sample.size", "200"); <5>
+config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis", "100"); <6>
 [...]
 ----
+====
+<1> The value must be `phi-accrual`.
+<2> xref:deadline-detector.adoc#heartbeat-interval[Interval] at which member heartbeat messages are sent to each other.
+<3> Timeout which defines when a cluster member is suspected because it has not sent any heartbeats.
+Since _Phi Accrual Failure Detector_ is adaptive to network conditions, a much lower `hazelcast.max.no.heartbeat.seconds` can be defined than
+_Deadline Failure Detector_'s xref:deadline-detector.adoc#heartbeat-timeout[timeout].
+<4> The phi threshold for suspicion. After calculated phi exceeds this threshold, a member
+is considered as unreachable and marked as suspected. A low threshold allows to
+detect member crashes/failures faster but can generate more mistakes and cause
+wrong member suspicions. A high threshold generates fewer mistakes but is slower
+to detect actual crashes/failures. `phi = 1` means likeliness that we will make a mistake is about `10%`. The likeliness
```

Comment:
If you set `phi = 1`, the possibility of a incorrectly-identified failure is around 10%. If you set `phi = 2`, this falls to around 1%, and setting `phi = 3` reduces this further to around 0.1%. The likelihood of incorrectly-identified failures continues to reduce in this way as you increase the value.